Project 1/shortstoryalt.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro|Spectral&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <h1>The King's Bodyguard</h1>
+    <h2> by Franz Xaver von Schönwerth</h2>
+    <h2> April 30th 2015</h2>
+  </header>
+  <main>
+    <p>
+      A king appointed the most handsome of all his soldiers as his bodyguard. One day he dispatched him to
+      a rich
+      nobleman, who made a point of protecting his wondrously beautiful daughter from soldiers by telling her:
+      "Anna,don't exchange a single word with
+      a soldier!" But this soiler happened to like the beautiful young woman, and Anna happened to like him as well.
+      Before long, they were on intimate terms with each other, and the bodyguard asked the nobleman for his
+      daughter's
+      hand. The girls father was
+      furious when he heard the request, for Anna was his only daughter, and the bodyguard was sure to be nothing but
+      a
+      fortune hunter.
+    </p>
+
+    <p> Some time passed, and Anna gave birth to a sweet little prince. The nobleman asked the guard: "if you reach the
+      top of the mountain made of glass and steal three golden feathers from the tail of the dragon there, I will let
+      you marry my daughter."</p>
+
+    <p>the guard began the long march to the mountain. on his way he passed through a kingdom and had to report the
+      purpose of his visit to the local ruler. When the king learned that the young man was on his way to the dragon, he
+      told him to ask about the golden
+      chain that he had lost some time ago. The soldier would not be able to travel through his kingdom unless he put
+      that question to the dragon.</p>
+
+    <p>A year later he was given a task by another king: "I have a fig tree that bears no fruit. Ask the dragon why!
+    </p>
+
+
+    <aside class="pquote">
```

Comment:
You should apply this class to the `blockquote` instead -- the `aside` element is not allowed to be inside the `main` element. The MDN docs don't reference this, but having `aside` inside of `main` is an accessibility problem; you can read more about this [here](https://dequeuniversity.com/rules/axe/3.3/landmark-complementary-is-top-level).